src/gsCore/gsMultiBasis.h

```diff
@@ -345,15 +345,17 @@ class gsMultiBasis : public gsFunctionSet<T>
     /// grid functions.
     ///
     /// For computing the transfer matrix (but not for refinement), the \a boundaryConditions and
-    /// the \a assemblerOptions have to be provided
+    /// the \a assemblerOptions have to be provided. By deault, the boundary conditions for
+    /// component 0 are chosen. Use the parameter comp to choose another one.
     ///
     /// \sa gsMultiBasis::uniformRefine
     void uniformRefine_withTransfer(
         gsSparseMatrix<T, RowMajor>& transfer,
         const gsBoundaryConditions<T>& boundaryConditions,
         const gsOptionList& assemblerOptions,
         int numKnots = 1,
-        int mul = 1
+        int mul = 1,
+        index_t comp = 0
```

Comment:
As in:
void gsMultiBasis::getMapper(...,int unk,..)

it should be called "unk" (unknown) instead of component..
Terminology: For a system with eg. 2 unknowns u,p, each unknown can have dimension>1, ie. component 0,1...